cypress/e2e/pageObjects/OrangeHRMAddEmployee/dataUtils.ts

```diff
@@ -52,19 +48,39 @@ UpdateEmployee(UpdateAPIBody: UpdateAPIBody,empNumber: string){
         url: 'https://opensource-demo.orangehrmlive.com/web/index.php/api/v2/pim/employees/'+empNumber+'/personal-details',
         body:
         UpdateAPIBody
-      }).then((response) => {
-       expect(response.status).to.equal(200);
-    });   
+      });  
 }
 
-checkUpdatedPersonalDerails(UpdateAPIResponse: UpdateAPIResponse, empNumber: string){
+searchOnEmployeeByName(name: string){
   cy.request({
-      method: 'GET',
-      url: 'https://opensource-demo.orangehrmlive.com/web/index.php/api/v2/pim/employees/'+empNumber+'/personal-details',
-    }).then((response) => {
-      expect(response.status).to.equal(200);
-      expect(response.body.data).to.equal(UpdateAPIResponse);
-    });
+    method: 'GET',
+    url: 'https://opensource-demo.orangehrmlive.com/web/index.php/api/v2/pim/employees?limit=50&offset=0&model=detailed&nameOrId='+name+'&includeEmployees=onlyCurrent&sortField=employee.firstName&sortOrder=ASC',
+  }).then((response) => {
+    for(let i=0; i<response.body.meta.total; i++){
+    expect(response.body.data[i].firstName).to.include(name);
+    }
+  });
+}
+
+ searchOnEmployeeByID(id :string){
+  return cy.request({
+    method: 'GET',
+    url: 'https://opensource-demo.orangehrmlive.com/web/index.php/api/v2/pim/employees?limit=50&offset=0&model=detailed&employeeId='+id+'&includeEmployees=onlyCurrent&sortField=employee.firstName&sortOrder=ASC',
+  }).then((response) => {
+    expect(response.body.data[0].employeeId).to.equal(id);
+    expect(response.body.meta.total).to.equal(1);
+    return response.body;
```

Comment:
you can just return this from this API **return response.body.data**
No need for line 70 and 71